components/eventing-controller/testing/bebmock.go

```diff
@@ -24,8 +24,9 @@ import (
 )
 
 const (
-	TokenURLPath     = "/auth"
-	MessagingURLPath = "/messaging"
+	TokenURLPath           = "/auth"
+	MessagingURLPath       = "/messaging"
+	EventMeshMockKeyPrefix = "/messaging/events/subscriptions"
```

Comment:
(nit) Preferable to be moved in the only place where it is used.

Reply:
updated